src/cmd.ts

```diff
@@ -297,7 +297,25 @@ program
 		});
 		await db.destroy();
 	});
-
+program
+	.command("clear-indexer-failures")
+	.description("Clear the cached details of indexers (failures and caps)")
+	.action(async () => {
+		console.log(
+			"\nIf you've received a '429' (rate-limiting), continuing to hammer",
+			"your indexers may result in negative consequences.",
+		);
+		console.log(
+			"If you have to do this more than once in a short",
+			"period of time, you have bigger issue that need to be addressed.",
+		);
+		await db("indexer").update({
+			active: true,
```

Comment:
should we really set active to true? we don't do it in line 291

Reply:
oh, now that you mention it i guess removed trackers would become active then.....

ill fix this.